### ISSUE TITLE
Data World Access

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#76]
As of [v0.1.3](https://github.com/timtrice/rrricanes/releases/tag/v0.1.3) there are two methods of extracting storm data: any of the `get` functions and `load_storm_data`.

The `get` functions scrape the NHC archives which [as document](https://github.com/timtrice/rrricanes/issues/72) [extensively](https://github.com/timtrice/rrricanes/issues/71)  [can fail](https://github.com/timtrice/rrricanes/issues/66) for a number of reasons.

The `get` functions should only be for updating and inserting into datasets. Which really means that users **shouldn't** need them unless absolutely necessary. `load_storm_data` should be the fallback.

This requires that the dataset repo, [rrricanesdata](https://github.com/timtrice/rrricanesdata) must be updated immediately with active storm data. I can resolve this with no issues; it's just a matter of writing the script and scheduling it. 

I am also going to add a third option through data.world. This will require users to have a data.world API key. data.world allows no more than 100 files per dataset with total storage of 500MB ([source](https://help.data.world/support/solutions/articles/14000039826-what-are-the-size-restrictions-for-data-world-)). 

The current tidied datasets (`adv`, `fcst`, `fcst_wr`, `storms`, and `wr`) are around 10MB in size; nowhere close to the limit. So there should be plenty of room to add `discus`, `posest`, `public` and `updates`. Hopefully, `recon` data which is the next priority can be added as well but if it requires a pay-tier that will not be an issue, personally. 

I considered adding GIS data but I have not had nearly the issues I've had with scraping data so, at this point of time, it will be left as is.

Additionally, using data.world will not require additional functionality added into the package so that will help keep it clean. However, users will be encouraged to install the `dwapi` package via CRAN. 

Therefore, `load_storm_data` will be modified to import the tidied datasets rather than it's current setup (by year and product). It will continue to pull from `rrricanesdata` which will serve as the backup to data.world. 

data.world will be an "enhanced querying" tool so that users can create more complex SQL queries to isolate only the data they wish to examine without cluttering their R environment. 

data.world will be setup to immediately update current datasets on any changes in the GitHub repo. So only GitHub will need to be updated as new storms develop and new datasets are added (`public`, `discus`, `posest`, `updates`).

tldr:

`rrricanesdata` will store tidied data files (not grouped by year, basin as is currently). `load_storm_data` will return full datasets as requested by user from this repo. 

Users will be given documentation on accessing data.world with examples of querying data with SQL. No additional functionality is needed within the package.

The `get` functions will not be deprecated but considered a "last resort" to obtain data.

These changes will be implemented in release 0.2.0.
